tests/csm/rest/test_capacity_quota.py

```diff
@@ -2688,3 +2688,34 @@ def test_43862(self):
                 assert error.response['Error']['Code'] == "QuotaExceeded", \
                                       "Second overwrite passed"
         self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-44411')
+    def test_44411(self):
+        """
+        Verify user capacity after recursively deleting all objects in bucket
+        """
+        test_case_name = cortxlogging.get_frame()
+        test_cfg = self.csm_conf["test_44411"]
+        max_size = test_cfg["max_size"]
+        random_size = self.csm_obj.random_gen.randrange(1, max_size)
+        num_objects = test_cfg["num_objects"]
+        random_objects = self.csm_obj.random_gen.randrange(1, num_objects)
+        self.log.info("Step 1:Create and upload objects in a bucket")
+        for _ in range(0, random_objects):
```

Comment:
Can u log to print which Iteration is going on?

Reply:
done